doc/netplan.md

```diff
@@ -1377,6 +1390,111 @@ Example:
+     :    Takes a boolean. When true, enables dynamic MAC learning to
+          discover remote MAC addresses.
+
+     ``fdb-ageing`` (scalar)
+     :    The lifetime of Forwarding Database entry learnt by the kernel, in seconds.
+
+     ``max-fdb-entries`` (scalar)
+     :    Configures maximum number of FDB entries.
+
+     ``reduce-arp-proxy`` (scalar)
+     :    Takes a boolean. When true, bridge-connected VXLAN tunnel endpoint answers
+          ARP requests from the local bridge on behalf of remote Distributed Overlay
+          Virtual Ethernet (DOVE) clients. Defaults to false.
+
+     ``l2-miss-notification`` (scalar)
+     :    Takes a boolean. When true, enables netlink LLADDR miss notifications.
+
+     ``l3-miss-notification`` (scalar)
+     :    Takes a boolean. When true, enables netlink IP address miss notifications.
+
+     ``route-short-circuit`` (scalar)
+     :    Takes a boolean. When true, route short circuiting is turned on.
+
+     ``udp-checksum`` (scalar)
+     :    Takes a boolean. When true, transmitting UDP checksums when doing VXLAN/IPv4 is turned on.
+
+     ``udp6-zero-checksum-tx`` (scalar)
+     :    Takes a boolean. When true, sending zero checksums in VXLAN/IPv6 is turned on.
+
+     ``udp6-zero-checksum-rx`` (scalar)
+     :    Takes a boolean. When true, receiving zero checksums in VXLAN/IPv6 is turned on.
+
+     ``remote-checksum-tx`` (scalar)
+     :    Takes a boolean. When true, remote transmit checksum offload of VXLAN is turned on.
+
+     ``remote-checksum-rx`` (scalar)
+     :    Takes a boolean. When true, remote receive checksum offload in VXLAN is turned on.
+
+     ``group-policy-extension`` (scalar)
+     :    Takes a boolean. When true, it enables Group Policy VXLAN extension security label
+          mechanism across network peers based on VXLAN. Defaults to false.
+
+     ``generic-protocol-extension`` (scalar)
+     :    Takes a boolean. When true, Generic Protocol Extension extends the existing VXLAN protocol
+          to provide protocol typing, OAM, and versioning capabilities. Defaults to false.
+
+     ``destination-port`` (scalar)
+     :    Configures the default destination UDP port. If the destination port is not specified
+          then Linux kernel default will be used. Set to 4789 to get the IANA assigned value.
+
+     ``port-range`` (scalar)
+     :    Configures the source port range for the VXLAN. The kernel assigns the source UDP port
+          based on the flow to help the receiver to do load balancing. When this option is not set,
+          the normal range of local UDP ports is used.
+
+     ``flow-label`` (scalar)
+     :    Specifies the flow label to use in outgoing packets. The valid range is 0-1048575.
+
+     ``ip-do-not-fragment`` (scalar)
+     :    Allows setting the IPv4 Do not Fragment (DF) bit in outgoing packets. Takes a boolean value.
+          When unset, the kernel's default will be used.
+
+     ``independent`` (scalar)
```

Comment:
I'd like to avoid this setting, as it is very backend (systemd-networkd) specific. We already have this information if NetplanNetDefinition->vxlan_link is set and can set the value implicitly. No need to expose it to the user via YAML

Reply:
Sure - I removed this from the PR.